data/docs/metrics-management/docker-container-metrics.mdx

```diff
@@ -0,0 +1,199 @@
+
+- CPU
+- Memory
+- Network
+- Disk
+
+### CPU Metrics
+
+The dashboard contains a panel that shows the percentage of CPU usage (`container_cpu_utilization`) of the containers:
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/cpu_percent.webp" width="80%" alt="CPU Usage"/>
+    <figcaption><i>CPU Usage Panel</i></figcaption>
+</figure>
+
+### Memory Metrics
+
+There are couple of panels related to memory. The first panel shows the percentage of memory usage (`container_memory_percent`) of the containers:
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/memory_percent.webp" width="80%" alt="Memory Usage"/>
+    <figcaption><i>Memory Usage Panel</i></figcaption>
+</figure>
+
+The second panel related to memory shows the memory limits (`container_memory_usage_limit`) and usage (`container_memory_usage_total`) of the containers. This is useful to understand if the containers are hitting the memory limits and might need more resources:
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/memory_limits.webp" width="80%" alt="Memory Limits"/>
+    <figcaption><i>Memory Limits Panel</i></figcaption>
+</figure>
+
+### Network Metrics
+
+There are two panels that display Network Bytes received and sent by the containers. The panels uses `container_network_io_usage_rx_bytes` and `container_network_io_usage_tx_bytes` metrics respectively:
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/network_bytes.webp" width="80%" alt="Network Bytes"/>
+    <figcaption><i>Network Bytes Panels</i></figcaption>
+</figure>
+
+Additionally, there's a panel that shows the number of packets dropped by the containers, which uses `container_network_io_usage_rx_dropped` and `container_network_io_usage_tx_dropped` metrics:
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/network_packets_dropped.webp" width="80%" alt="Network Packets Dropped"/>
+    <figcaption><i>Network Packets Dropped Panels</i></figcaption>
+</figure>
+
+### Disk Metrics
+
+The dashboard also includes a "Block IO" panel based on `container.blockio.io_serviced_recursive` metric that shows the number of IOs (bio) issued to the block device per second as well as the type of IO operations (read/write):
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/metrics/docker/block_io.webp" width="80%" alt="Block IO"/>
+    <figcaption><i>Block IO Panel</i></figcaption>
+</figure>
+
+## Conclusion
+
+Following this guide, you should have a working setup for collecting Docker container metrics using OTel Collector and visualizing them in SigNoz.
+
+You can find the full list of metrics collected by the `docker_stats` receiver and available for querying in dashboards [in the official receiver documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/documentation.md).
```

Comment:
Add this as the final sentence after Disk Metrics under the Visualizing Docker Container Metrics section.